res/demo/app.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "runtime": {
-        "arguments": "--v=1 --enable-crash-reporting",
+        "arguments": "--security-realm=of-notifications-service",
```

Comment:
Why would the removed arguments be ignored, but not these?

Reply:
Discussed offline